ONSdigital/dp-frontend-filter-dataset-controller
# Remove dataset preview

### What

- Added feature flag `ENABLE_DATASET_PREVIEW` checking and logic to prevent api calls nad mapping logic. 
- Pass feature flag value to page model, to be passed onto renderer 

### How to review

- Pull https://github.com/ONSdigital/dp-frontend-renderer/pull/329
- Run `dp-frontend-filter-dataset-controller` with `ENABLE_DATASET_PREVIEW=false` set. 
- Filter a dataset and proceed to the output page. **No** preview should show

### Who can review

Anyone
